indico/web/templates/forms/principal_list_widget.html

```diff
@@ -2,8 +2,13 @@
 
 
 {% block html %}
+    {% if acl %}
+        <a href="#" class="enable-acl-link hidden" data-qtip-oldtitle="Protected mode will be enabled">
```

Comment:
I think `data-qtip-oldtitle` is not needed and the `title="{% trans %}Protected mode will be enabled{% endtrans %}"` should be enough, but verify.